Source/CesiumRuntime/Public/CesiumLoadedTile.h

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020-2024 CesiumGS, Inc. and Contributors
+
+#pragma once
+
+#include "Cesium3DTilesSelection/Tile.h"
+#include "CesiumModelMetadata.h"
+#include "CesiumPrimitiveFeatures.h"
+#include "CesiumPrimitiveMetadata.h"
+
+#include "Math/Vector.h"
+#include "UObject/ObjectMacros.h"
+
+#include "CesiumLoadedTile.generated.h"
+
+class ACesium3DTileset;
+class UStaticMeshComponent;
+
+UINTERFACE()
+class UCesiumLoadedTile : public UInterface {
+  GENERATED_BODY()
+};
+/** Access to the data loaded when a tile has been selected for display by the
+ * tileset.
+ */
+class ICesiumLoadedTile {
+  GENERATED_BODY()
+public:
+  /** The tile identifier: this is informational only, as there is no
+   * guarantee of uniqueness */
+  virtual const Cesium3DTilesSelection::TileID& GetTileID() const = 0;
+  /** The glTF model from which the tile data was obtained. */
+  virtual const CesiumGltf::Model* GetGltfModel() const = 0;
+  /** The scaling factor to be applied (component-wise multiplication) to glTF
+   * vertices of this tile's model to obtain the values represented in their
+   * matching mesh component (see @{link
+   * UCesiumLoadedTilePrimitive::GetMeshComponent}). */
+  virtual FVector GetGltfToUnrealLocalVertexPositionScaleFactor() const = 0;
+  /** The tileset actor which the tile belongs to. */
+  virtual ACesium3DTileset& GetTilesetActor() = 0;
+  /** The blueprint-accessible wrapper for metadata contained in the tile's glTF
+   * model. */
+  virtual const FCesiumModelMetadata& GetModelMetadata() const = 0;
+};
+
+// Not merged with ICesiumPrimitive because this is Public whereas
+// ICesiumPrimitive is Private
+UINTERFACE()
+class UCesiumLoadedTilePrimitive : public UInterface {
+  GENERATED_BODY()
+};
+/** Access to the data loaded for a given glTF primitive of an {@link
+ * ICesiumLoadedTile}.
+ */
+class ICesiumLoadedTilePrimitive {
+  GENERATED_BODY()
+public:
+  /** The {@link ICesiumLoadedTile} the primitive belongs to. */
+  virtual ICesiumLoadedTile& GetLoadedTile() = 0;
+  /** The Unreal mesh component built to represent the glTF primitive. */
+  virtual UStaticMeshComponent& GetMeshComponent() = 0;
```

Comment:
Should this be a pointer, in order to allow for the posibility that a primitive is not represented as a static mesh component?